connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -648,6 +648,14 @@ private WorkerTask buildWorkerTask(ClusterConfigState configState,
         // These settings will execute infinite retries on retriable exceptions. They *may* be overridden via configs passed to the worker,
         // but this may compromise the delivery guarantees of Kafka Connect.
         producerProps.put(ProducerConfig.MAX_BLOCK_MS_CONFIG, Long.toString(Long.MAX_VALUE));
+        // By default, producers that are instantiated and used by Connect have idempotency disabled even after idempotency became
+        // default for Kafka producers. This is chosen to avoid breaking changes when Connect contacts Kafka brokers that do not support
```

Comment:
Maybe rephrase to be more positive? Like:
```suggestion
        // default for Kafka producers. This ensures Connect continues to work with many Kafka broker versions, including older brokers that do not support
```
If you make a change like this, please change the other places to be consistent.

Reply:
Thanks for the suggestion. Added it in all places